# roomba_dynamixel_controller's parameter
execution_time: 0.4

# tf parameter
dynamixel_name: dynamixel
dynamixel_frame_id: dynamixel
base_link_frame_id: base_link
dynamixel_x: 0.0
dynamixel_y: 0.0
dynamixel_z: 0.633
dynamiel_roll: 0.0
dynamiel_pitch: 0.0